api/AbortController.json

```
@@ -80,12 +90,26 @@
             "opera_android": {
               "version_added": "47"
             },
-            "safari": {
-              "version_added": "11.1"
-            },
-            "safari_ios": {
-              "version_added": "11.3"
-            },
+            "safari": [
+              {
+                "version_added": "12.1"
+              },
+              {
+                "version_added": "11.1",
+                "partial_implementation": true,
+                "notes": "Even though <code>window.AbortController</code> is defined, it doesn't really abort <code>fetch</code> requests. See <a href='https://webkit.org/b/174980'>bug 174980</a>."
+              }
+            ],
+            "safari_ios": [
+              {
+                "version_added": "12.2"
+              },
+              {
+                "version_added": "11.3",
+                "partial_implementation": true,
+                "notes": "Even though <code>window.AbortController</code> is defined, it doesn't really abort <code>fetch</code> requests. See <a href='https://webkit.org/b/174980'>bug 174980</a>."
+              }
+          ],
```

Whoops, looks like you've got a little styling error!

```suggestion
            ],
```

---

> Whoops, looks like you've got a little styling error!

oops yeah, my fault for not taking time to do `npm run test` — thanks for catching it